doc/api/n-api.md

```diff
@@ -34,13 +34,53 @@ properties:
   handling section [Error Handling][].
 
 The N-API is a C API that ensures ABI stability across Node.js versions
-and different compiler levels. However, we also understand that a C++
-API can be easier to use in many cases. To support these cases we expect
-there to be one or more C++ wrapper modules that provide an inlineable C++
-API. Binaries built with these wrapper modules will depend on the symbols
-for the N-API C based functions exported by Node.js. These wrappers are not
-part of N-API, nor will they be maintained as part of Node.js. One such
-example is: [node-addon-api](https://github.com/nodejs/node-addon-api).
+and different compiler levels. **However, a C++ API can be easier to use
+in many cases.** In order to support using C++ the project maintains a
```

Comment:
Do we really need to add bold text here? (No strong opinion other than we have a tendency to overuse bold and italics so it always seems worth asking.)

Reply:
If you really want people to take notice of the sentence, make it more direct by getting rid of qualifiers like "However" and "in many cases" so it's just: 

> A C++ API can be easier to use.

Maybe even set it off as its own single-sentence paragraph if it's that important.

Reply:
`In order to` -> `To`

Reply:
Comma needed after `C++`